### PR TITLE
Fixed PHPC-357: The exception for "invalid namespace" does not list the broken name

### PR DESCRIPTION
**Pull request by derickr**
The test case is copied over from HHVM

### CODE REVIEW

File: tests/manager/bug0357.phpt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--TEST--
+PHPC-357: The exception for "invalid namespace" does not list the broken name
+--FILE--
+<?php
+$m = new MongoDB\Driver\Manager("mongodb://localhost:44444");
```
Comment (bjori) [inline tests/manager/bug0357.phpt]: This test is missing the standard skipifs -- and should be using the constants.

Comment (derickr) [inline tests/manager/bug0357.phpt]: Happy to add the skip-ifs, but if you mean by "constants" the URI - then no, as I needed one that is likely not to connect to anything ever. I don't have MongoDB on 44444 :-)

Comment (bjori) [inline tests/manager/bug0357.phpt]: It should never connect to anything as the exception should be thrown before attempting to connect to anything.

File: tests/manager/bug0357.phpt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+try {
+	$m->executeQuery( 'demo', $c );
+	echo "Expected exception not thrown\n";
+}
+catch ( InvalidArgumentException $e )
+{
+	echo $e->getMessage(), "\n";
+}
+?>
```
Comment (bjori) [inline tests/manager/bug0357.phpt]: please add (after the closing ?> and before --EXPECTF--)
```
===DONE===
<?php exit(0); ?>
```

File: tests/manager/bug0357.phpt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+require_once __DIR__ . "/../utils/basic.inc";
+$m = new MongoDB\Driver\Manager("mongodb://localhost:44444");
+
+$c = new MongoDB\Driver\Query( [] );
+
+try {
+	$m->executeQuery( 'demo', $c );
+	echo "Expected exception not thrown\n";
+}
+catch ( InvalidArgumentException $e )
```
Comment (bjori) [inline tests/manager/bug0357.phpt]: we do have throws() wrapper...

throws(function() use ($m) {
    $c = new MongoDB\Driver\Query( [] ); 
    $m->executeQuery( 'demo', $c );
}, "MongoDB\\Driver\\InvalidArgumentException");

Comment (bjori) [inline tests/manager/bug0357.phpt]: oh, you are asserting the exact message... nevermind

**Comment by jmikola** [comment]
Rebased in https://github.com/10gen-labs/mongo-php-driver-prototype/commit/dead06a80bdc9905838f8cc0ce56407bafe45528, revisions in https://github.com/10gen-labs/mongo-php-driver-prototype/commit/e61b67809eedbc1893ed173677269d7440d49a9c, and merged in https://github.com/10gen-labs/mongo-php-driver-prototype/commit/2436e158c9a7e8d6a0bf2eba5edfb06ff381123e.

File: tests/manager/bug0357.phpt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+--TEST--
+PHPC-357: The exception for "invalid namespace" does not list the broken name
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+$m = new MongoDB\Driver\Manager("mongodb://localhost:44444");
```
Comment (jmikola) [inline tests/manager/bug0357.phpt]: I ended up relocating this test to the `standalone/` directory, and also replaced this with the `STANDALONE` constant. I assume `localhost:44444` wasn't purposeful here.

File: tests/manager/bug0357.phpt (hunk 3 above)
Comment (jmikola) [inline tests/manager/bug0357.phpt]: Done in my revised commit.

File: tests/manager/bug0357.phpt (hunk 2 above)
Comment (jmikola) [inline tests/manager/bug0357.phpt]: Done in my revised commit.

**Pull request closed without merging**